python/strings_udf/strings_udf/tests/test_string_udfs.py

```diff
@@ -50,14 +58,21 @@ def run_udf_test(data, func, dtype):
     and then assembles the result back into a cuDF series before
     comparing it with the equivalent pandas result
     """
-    dtype = np.dtype(dtype)
-    cudf_column = cudf.core.column.as_column(data)
-    str_view_ary = to_string_view_array(cudf_column)
+    if dtype == "str":
+        output_ary = rmm.DeviceBuffer(size=len(data) * udf_string.size_bytes)
```

Comment:
While we're here I would change the variable names and get rid of the `_ary` suffixes. No need to introduce Hungarian notation to our codebase.